apps/storybook.namekit.io/global.css

```
@@ -2,3 +2,12 @@
   color: red;
   background: white;
 }
+
+.huge-font-size {
+  font-size: 80px;
+}
+
+.beautiful-text {
```

```suggestion
.colorful-text {
```

Better to make this more specific.